macros/adapters/default/page_views/snowplow_web_events.sql

```diff
@@ -122,7 +122,7 @@ prep as (
         ev.os_name,
         ev.os_family,
         ev.os_manufacturer,
-        ev.os_timezone,
+        replace(ev.os_timezone, '%2F', '/')
```

Comment:
I think you're going to want to alias this as `os_timezone` :)

Reply:
🤦‍♂️ Haha - Coffee required.